src/main/scala/uk/gov/hmrc/apiplatform/addapi/UpsertApiHandler.scala

```diff
@@ -127,16 +127,6 @@ class UpsertApiHandler(override val apiGatewayClient: ApiGatewayClient,
       NoCloudWatchLogging,
       AccessLogConfiguration(AccessLogFormat, environment("access_log_arn")))
 
-    associateWebACL(restApiId)
     usagePlanService.addApiToUsagePlans(restApiId, titleWithoutVersion)
   }
-
-  private def associateWebACL(restApiId: String)(implicit logger: LambdaLogger): Unit = {
-    val stageArn: String = s"arn:aws:apigateway:${environment("AWS_REGION")}::/restapis/$restApiId/stages/current"
-    val webAclId: String = environment("waf_acl_id")
```

Comment:
I'm wondering when we want to stop setting this env var.  We could do it now, as the lambda no longer needs it, or when we actually remove the legacy waf - I'm assuming terraform will force us to remove it then as it will no longer have an id.

It's defined here and I think it will be cleaner to remove it when we remove the legacy waf, keeping all the terraform changes to one ticket. What do you think?

https://github.com/hmrc/webops-terraform/blob/3b85da9b25213deda6d2aa5c5ac65a7f28a1c7aa/components/api_manager/main.tf#L396

Reply:
I initially thought this `waf_acl_id` would be automatically provided by AWS but then I realised that it's in our control and defined through the `webops-terraform` repo.
We've discussed this separately and agreed that no longer setting the ID should be done as part of the work to remove the WAF from Terraform.